packages/inventory-insights/src/Insights.js

```diff
@@ -331,7 +333,9 @@ class InventoryRuleList extends Component {
                             <RemediationButton
                                 isDisabled={ this.getSelectedItems(rows).length === 0 }
                                 dataProvider={ this.processRemediation }
-                                onRemediationCreated={ result => this.props.addNotification(result.getNotification()) } />
+                                onRemediationCreated={ result => this.props.addNotification(result.getNotification()) } >
+                                <AnsibeTowerIcon size='sm' color={ global_BackgroundColor_100.value } /> Remediation
```

Comment:
Can you make this into a new style in `.scss` file? That way we don't have to import this const from pf styles.

Reply:
So this const comes from the `reac-tokens`, and we're already using it in another place...  🤔 `global_success_color_200` yah want me to move both of those into the scss?